content/manuals/engine/release-notes/27.md

```diff
@@ -27,6 +27,27 @@ For more information about:
 
 Release notes for Docker Engine version 27.4 releases.
 
+### 27.4.1
+
+{{< release-date date="2024-12-16" >}}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 27.4.1 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A27.4.1)
+- [moby/moby, 27.4.1 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A27.4.1)
+
+
+#### Bug fixes and enhancements
+
+- Fix excessive memory allocations when OTel is not configured. [moby/moby#49079](https://github.com/moby/moby/pull/49079)
+- `docker info` and the corresponding `GET /info` API endpoint no longer include warnings when `bridge-nf-call-iptables` or `bridge-nf-call-ip6tables` are disabled at the daemon is started. The `br_netfilter` kernel module is now attempted to be loaded when needed, which made those warnings inaccurate. [moby/moby#49090](https://github.com/moby/moby/pull/49090)
+- Attempt to load kernel modules, including `ip6_tables` and `br_netfilter` when required, using a method that is likely to succeed inside a Docker-in-Docker container. [moby/moby#49043](https://github.com/moby/moby/pull/49043)
+
+#### Packaging updates
+
+- Update Compose to [v2.32.0](https://github.com/docker/compose/releases/tag/v2.32.0). [docker/docker-ce-packaging#1126](https://github.com/docker/docker-ce-packaging/pull/1126)
```

Comment:
This will be compose v2.32.1 now;

- https://github.com/docker/docker-ce-packaging/pull/1130

update to buildx is also coming later (if all goes well)